timeboost-crypto/src/sg_encryption.rs

```diff
@@ -170,9 +178,10 @@ where
         comb_key: &Self::CombKey,
         dec_shares: Vec<&Self::DecShare>,
         ciphertext: &Self::Ciphertext,
+        aad: &Self::AssociatedData,
     ) -> Result<Self::Plaintext, ThresholdEncError> {
         let committee_size: usize = committee.size.get();
-        let threshold = committee.threshold().get() + 1;
+        let threshold = committee.threshold().get();
```

Comment:
after seeing how sequencer is using this code, i realize that my change (introduced in [9870e7f](https://github.com/EspressoSystems/timeboost/pull/340/commits/9870e7f062d17014ed25b1d8aaff024ad307f1e9)) is probably wrong.

when i made the change, i was looking at the scheme impl in isolation, and assume we can take the "threshold" on its standard `(t, N)`-threshold definition. but here we need to set a value congruent with consensus's threshold. so `t+1` makes more sense. 

I will try to explain more in an issue and fix my own mess.